src/shared/utilities/promptSetting.ts

```diff
@@ -0,0 +1,37 @@
+/*!
```

Comment:
the codebase already has `src/shared/settingsConfiguration.ts`, what is the reason for adding this separate module ?